request/consumers/sickbeard.py

```diff
@@ -89,16 +89,16 @@ def help(self, args):
 
     def get_all(self, args):
         """Get a list of all movies on the server"""
-        data = self.make_request('shows')
+        data = self.make_request('?cmd=shows')
```

Comment:
Instead of this, I'd suggest adding a `Sickbeard.make_sickbeard_request` method:

    def make_sickbeard_request(self, method, query=None, body=None):
        query.update({'cmd': method})
        self.make_request(method=None, query=query, body=body)

and then make this

    data = self.make_sickbeard_request('shows')